### PR TITLE
Revision of Introduction draft

### PR DESCRIPTION
**Pull request by mickohanlon23**
Edits and comments from review of Introduction section by Patrick Randall and MJ Sullivan. 

Patrick and MJ both sent their review via a Word Doc. I added their suggested edits in-line, and I added their comments as comments to this PR. 

@radamson Please review, and add any commits based upon the comments as you see fit.

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -5,18 +5,30 @@ slug: /
 
 # Introduction
 
-This site provides documentation to assist in understanding, testing and use of vaccine credentials with a federated PKI.
+This guide provides documentation to assist in understanding, testing and the use of vaccine credentials with a federated Public Key Infrastructure (PKI).
 
-## Background
+## Background & Purpose
```
Comment (mickohanlon23) [inline docs/intro.md:21]: From Patrick:

> Included under Vision in the [G20 Pilot Overview](https://mitre.sharepoint.com/:b:/r/sites/whoproposal/Shared%20Documents/WHO%20G20%20Trust%20Network/Reference/G20%20pilot%20presentations/20220429%20Day%201%20Presentation_Overview%20and%20Scope%20(Indonesia).pdf?csf=1&web=1&e=zaDyfd) is: “To foster successful digital transformation in health documentation protocol, to inform International Health Regulation Amendment.” Does this need to be referenced in the Introduction?

File: docs/intro.md [hunk 2]
```diff
@@ -5,18 +5,30 @@ slug: /
...
 
 In response to COVID-19, governments and organizations across the world have
 developed and adopted standards and technologies to create, present, and verify
-digital vaccination credentials. This guide details how to utilize a global
+digital vaccination credentials.
```
Comment (mickohanlon23) [inline docs/intro.md]: From Patrick:

> Above we call them "vaccine credentials." Suggest choosing one.

File: docs/intro.md [hunk 3]
```diff
@@ -5,18 +5,30 @@ slug: /
...
+
+This guide details how to utilize a global
 technical framework to allow interoperability of these credentials between
```
Comment (mickohanlon23) [inline docs/intro.md]: From Patrick - regarding the use of "interoperability": 
> Should this say "exchange"? The systems are interoperable, not the credentials.

File: docs/intro.md [hunk 4]
```diff
@@ -5,18 +5,30 @@ slug: /
...
-systems, while allowing domestic autonomy over their use. This global technical
+systems, while preserving domestic autonomy over their use. The global technical
 framework aims to harmonize multiple existing trust networks and platforms to
-provide a single federated mechanism which supports trusted exchange of
-COVID-19 credentials.
+provide a single federated mechanism that supports trusted exchange of
+COVID-19 credentials and facilitates safe international travel.
```
Comment (mickohanlon23) [inline docs/intro.md]: From Patrick:

> Some of the reference docs (like [20220519 Pilot Project - Project Charter_Indonesia_v1](https://mitre.sharepoint.com/:x:/r/sites/whoproposal/Shared%20Documents/WHO%20G20%20Trust%20Network/Reference/20220519%20Pilot%20Project%20-%20Project%20Charter_Indonesia_v1.xlsx?d=w64ec447db13d42df8519230c273779df&csf=1&web=1&e=MnLWxn)) mention COVID-19 certificates and "other trusted health documents." Do we want to mention that here, or limit to COVID-19?

File: docs/intro.md [hunk 5]
```diff
@@ -5,18 +5,30 @@ slug: /
...
+
+Criteria for establishing a system of trust include:
```
Comment (mickohanlon23) [inline docs/intro.md]: From Patrick:

> Included because the Codev wiki outline mentions them in this section.

**Review by nkarip — APPROVED** [review]

**Pull request merged**